packages/terra-slide-group/src/Slide.scss

```diff
@@ -17,6 +17,10 @@
     }
   }
 
+  .slide[aria-hidden] {
+    display: none;
```

Comment:
This style will hide the element immediately, causing the animations to not look quite right.  We're going to have to uplift to the react-transition-group ^2.0, which exposes handlers we can use to hide elements at the appropriate time.

Reply:
Changed made in [this commit](https://github.com/cerner/terra-core/pull/786/commits/cc81fa06ee69492644309d10f470f8686107c96b)

Reply:
I think we might be able to use `visibility: hidden;` instead of `display: none;` to prevent focus. This would play nicer with the animations.